easybuild/framework/easyblock.py

```diff
@@ -897,8 +895,13 @@ def obtain_file(self, filename, extension=False, urls=None, download_filename=No
                     source_urls = []
                 source_urls.extend(self.cfg['source_urls'])
 
-                # add https://sources.easybuild.io as fallback source URL
-                source_urls.append(EASYBUILD_SOURCES_URL + '/' + os.path.join(name_letter, location))
+                # add extra-source-urls CLI as either a first check, or a fallback.
+                self.log.warning("[extra_source_urls] %s", build_option('extra_source_urls'))
```

Comment:
Why are those warnings? I'd suggest to only log if there are any at all, as (at most) `info` and the individual urls as `debug`. And maybe only log the final value, not each intermediate

Reply:
Thanks, I added those in for me to make sure the values were making their way in the expected places. I removed these logs completely, the debug logs already print
```
== 2024-07-16 20:58:08,626 generaloption.py:1604 DEBUG generate_cmd_line adding extra-source-urls value ('https://sources.easybuild.io/',) default found. Not adding to args.
```